arwen/host/arwen.go

```diff
@@ -418,3 +423,8 @@ func (host *vmHost) AreInSameShard(leftAddress []byte, rightAddress []byte) bool
 func (host *vmHost) IsInterfaceNil() bool {
 	return host == nil
 }
+
+// SetRuntimeContext mocked method
```

Comment:
there is no mock here

Reply:
you can have an extended test context for this. look into testEconomicsHandler in elrond.go

Reply:
wrong copy / paste comment